jenkins_init.sh

```diff
@@ -32,12 +32,12 @@ check_wsl() {
     if [[ $(grep -i Microsoft /proc/version) ]]; then
         echo "Running on WSL"
         if ! command -v dos2unix &> /dev/null; then
-            if sudo -n true 2>/dev/null; then
-                echo "dos2unix not found, installing..."
-                sudo apt-get update && sudo apt-get install -y dos2unix
-            else
+            if [[ $(id -u) -ne 0 ]]; then
```

Comment:
It's a nice touch to install `dos2unix`, but it won't work for each and every distro.
Let's keep it like that for the time being.
The need for `dos2unix` can't be linked to a bad `git` configuration (it used to be my case) or to the use of a directory under `C:` (seen as `/mnt/c` instead of something that is on `/home` for example).

I have to reconfigure my PC thanks to something [like that](https://www.jetbrains.com/help/idea/how-to-use-wsl-development-environment-in-product.html#create_project_for_wsl).